tests/threshold_tools/test_threshold_tools.py

```diff
@@ -0,0 +1,182 @@
+"""
+Add test coverage for code that is not already tested in
+test_thresholds_returns.py or test_threshold_exceedence.py.
+"""
+
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+import scipy
+import xarray as xr
+
+from climakitae.explore import threshold_tools
+from climakitae.explore.threshold_tools import (
+    _calc_average_ess_gridded_data,
+    _calc_average_ess_timeseries_data,
+    _exceedance_count_name,
+    _get_distr_func,
+    _get_exceedance_events,
+    _get_fitted_distr,
+)
+
+
+class TestThresholdTools:
+
+    def test_calculate_ess(self):
```

Comment:
All class methods need at least a simple docstring, using the `"""` format, explaining the function's purpose.

Reply:
Added.